crypto/cipher_extra/aead_test.cc

```diff
@@ -1364,6 +1364,48 @@ TEST(AEADTest, TestGCMSIV256Change16Alignment) {
   free(encrypt_ctx_256);
 }
 
+TEST(AEADTest, TestMonotonicityCheck) {
+
+  static const uint8_t kEvpAeadCtxKey[32] = {0};
+
+  // Only the tls13() ciphers have monotonicity checks
+  struct {
+    const EVP_AEAD *cipher;
+    const size_t key_len;
+  } ctx[] = { { EVP_aead_aes_128_gcm_tls13(), 16},
+              { EVP_aead_aes_256_gcm_tls13(), 32} };
```

Comment:
Instead of hardcoding these lengths you can get them using `EVP_AEAD_key_length` and `EVP_AEAD_nonce_length`, and if it's just a list of aeads I'd rename it aeads_to_test.